http/headers/Activate-Storage-Access.json

```
@@ -7,7 +7,11 @@
           "spec_url": "https://privacycg.github.io/storage-access-headers/#activate-storage-access-header",
           "support": {
             "chrome": {
-              "version_added": "133"
+              "version_added": "133",
+              "notes": [
+                "From version 141 onwards, activating the `storage-access` permission via `Activate-Storage-Access` results in third-party cookies being sent only with requests to the calling embed's exact origin.",
+                "Before version 141, `storage-access` activation resulted in third-party cookies being sent with requests to the calling embed's site."
+              ]
```

This part is incorrect - in Chrome, `Activate-Storage-Access` was always origin-scoped, rather than site-scoped. (Designing `Activate-Storage-Access` was actually how I realized that `document.requestStorageAccess()` was doing the wrong thing, by being site-scoped.)
```suggestion
              "version_added": "133"
```